tests/integration/net/state/simple/peer/subscribe/with_add_test.go

```diff
@@ -337,10 +341,6 @@ func TestP2PSubscribeAddNone(t *testing.T) {
 				SourceNodeID: 1,
 				TargetNodeID: 0,
 			},
-			testUtils.SubscribeToCollection{
```

Comment:
todo: Either the name of this test should be changed slightly, or the test deleted (if this is tested elsewhere, which it might be).